src/v/cluster/topics_frontend.cc

```diff
@@ -241,6 +241,12 @@ ss::future<std::error_code> topics_frontend::do_update_data_policy(
 
 ss::future<std::error_code> topics_frontend::do_update_replication_factor(
   topic_properties_update& update, model::timeout_clock::time_point timeout) {
+    if (!_features.local().is_active(
+          features::feature::replication_factor_change)) {
+        throw std::runtime_error(
+          "Replication-factor change feature is disabled");
```

Comment:
This function returns a std::error_code, so we should set that rather than throwing a runtime error (runtime errors usually show up as unpleasant messages for the user).

Reply:
Totally agree, we should maybe introduce new error code like: `feature_not_supported`